frontend/amp/components/lib/Elements.tsx

```diff
@@ -4,7 +4,7 @@ import React from 'react';
 import { ImageBlockComponent } from '@frontend/amp/components/elements/ImageBlockComponent';
 import { InstagramBlockComponent } from '../elements/InstagramBlockComponent';
 
-export const AmpRenderer: React.SFC<{
+export const Elements: React.SFC<{
```

Comment:
Why did this need renaming? 
It took me quite a while to find this file after I rebased.